third_party/accessibility/ax/platform/ax_platform_node_win.cc

```diff
@@ -5266,7 +5266,6 @@ std::optional<EVENTID> AXPlatformNodeWin::MojoEventToUIAEvent(
   }
 }
 
-// static
```

Comment:
Need to make this a method as the property may depend on the role (e.g. for `kValueChanged`)